### ISSUE TITLE
feature request: distribute/autoinstall stratagus along with wargus

### ISSUE THREAD

**Issue by mimooh** [Wargus/wargus#162]
This was mentioned elsewhere and was initially consider a good idea. I think for end users it would be preferred. Would it be hard to implement? For debian stratagus is installed from a dependency in apt-get and it seems fine. Perhaps it could be done similarly for windows? The way some dlls are autoinstalled now.

**Comment by timfel** [comment]
The Stratagus installer is now included with Wargus, if Stratagus is not already installed, it is installed during the Wargus installation.

**Comment by Kintobor** [comment]
Why not just include the binary in the Wargus directory?

**Comment by timfel** [comment]
For modularity and being parallel to Debian.

**Comment by Kintobor** [comment]
For Windows the binary should be included in the directory so you can install multiple versions and have them portable.

**Comment by timfel** [comment]
Can you try the current installer? I've changed it so that stratagus.exe is shipped as @Kintobor requested

**Comment by mimooh** [comment]
Tim, I instructed my friend and hopefully he will report on how we are
about these winXP and related issues.

This weekend we played 6 players network game. One game was desyncing
after ~20 minutes, but the other 3 were fine. Not a big deal, we are
fine if it ocassionally desyncs, but as I said, out-of-sync is not that
easily removed completly :)

Also, right after I destroyed the whole village of one player, the game
hang for the rest of us. 

We can just leave it for now, until I have some more observations about
those. For now wargus is fine for us, so thanks again for your work :)
And -G option is just lovely! I will mention it on wargus site in
'benefits over warcraft' section.

-- 
Karol Kreński
